pages/linux/xsel.md

```diff
@@ -0,0 +1,16 @@
+# xsel
+
+> Selection manipulation tool.
+> For example xsel can copy STDIN to clipboard or print clipboard to STDOUT.
+
+- Copy output to clipboard:
+
+`echo 123 | xclip -bi`
+
+- Paste clipboard:
```

Comment:
How about "Print the clipboard to STDOUT"?

I actually might make some more examples to clarify things.